### PR TITLE
Generation for AKS

### PR DESCRIPTION
**Pull request by erich-wang**
Create an customize module for Azure AKS service

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Aks/readme.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+> `autorest`
+
+---
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+> Directives
+``` yaml
+metadata:
+  authors: Microsoft Corporation
```
Comment (markcowl) [inline src/Aks/readme.md]: Can we bring this in from ..\..\reeadme.azure.md?

File: src/Aks/readme.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+### General settings
+> Values
+
+``` yaml
+service-name: Aks
+powershell: true
+azure: true
+branch: master
+repo: https://github.com/Azure/azure-rest-api-specs/blob/$(branch)
+prefix: Az
```
Comment (markcowl) [inline src/Aks/readme.md]: I think all the metadata shoudl go in one single section, at the top

File: src/Aks/readme.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+title: AksClient
+
+directive:
+  - where:
+      subject: Operation
+    hide: true
+  - where: $.definitions.Identifier.properties
+    suppress: R3019
+  - where:
+      verb: New|Set|Remove|Get
```
Comment (markcowl) [inline src/Aks/readme.md]: Wy remove the Get and Remove cmdlets?

File: src/Aks/custom/RemoveAzAks.cs [hunk 4]
```diff
@@ -0,0 +1,604 @@
+using System;
+using System.Management.Automation;
+using System.Net.Http;
```
[the remark this answers is not in the document]
Comment (erich-wang) [reply src/Aks/custom/RemoveAzAks.cs]: Basically to change base class to align with other cmdlets.

File: src/Aks/examples/latest-2019-04-30/Get-AzBlockchainLocationConsortium.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
```
[the remark this answers is not in the document]
Comment (erich-wang) [reply src/Aks/examples/latest-2019-04-30/Get-AzBlockchainLocationConsortium.md]: will remove them

File: src/Aks/readme.md (hunk 1 above)
Comment (erich-wang) [reply src/Aks/readme.md]: Profiles are included ../readme.md, here I just use input files, they may mess up if using together. Will figure out better way after Ignite.

File: src/Aks/readme.md (hunk 2 above)
Comment (erich-wang) [reply src/Aks/readme.md]: will do it.

File: src/Aks/readme.md (hunk 3 above)
Comment (erich-wang) [reply src/Aks/readme.md]: need to change base class of get and remove, so hide here.

File: src/Aks/custom/ImportAzAksCredential.cs [hunk 6]
```diff
@@ -0,0 +1,684 @@
...
+                    }
+
+                    await ((IEventListener) this).Signal(Runtime.Events.CmdletBeforeAPICall);
+                    if (((IEventListener) this).Token.IsCancellationRequested)
+                    {
+                        return;
+                    }
+
+                    // get the client instance
+                    await ConfirmActionAsync(Force.IsPresent,
```
[the remark this answers is not in the document]
Comment (erich-wang) [reply src/Aks/custom/ImportAzAksCredential.cs]: I tried with -Confirm option, it works well. Does this address your concern?

File: src/Aks/docs/Get-AzAksAccessProfile.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+```
+
+## DESCRIPTION
+Gets the accessProfile for the specified role name of the managed cluster with a specified resource group and name.
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
+```powershell
+PS C:\> {{ Add code here }}
```
[the remark this answers is not in the document]
Comment (erich-wang) [reply src/Aks/docs/Get-AzAksAccessProfile.md]: add example for basic scenario

**Pull request merged**